module_utils/ca_common.py

```diff
@@ -2,15 +2,15 @@
 import datetime
 
 
-def generate_ceph_cmd(sub_cmd, args, user_key=None, cluster='ceph', user='client.admin', container_image=None, interactive=False):
+def generate_cmd(cmd: list, sub_cmd, args, user_key=None, cluster='ceph', user='client.admin', container_image=None, interactive=False):  # noqa: E501
```

Comment:
```suggestion
def generate_cmd(cmd, sub_cmd, args, user_key=None, cluster='ceph', user='client.admin', container_image=None, interactive=False):  # noqa: E501
```

(even though it would be nice to add type annotation...)